package.json

```diff
@@ -17,18 +17,16 @@
     "axios": "^0.21.1",
     "bootstrap": "4.5.3",
     "node-sass": "4.14.1",
-    "react": "^16.3.1",
+    "react": "^17.0.1",
     "react-bootstrap": "1.4.0",
-    "react-dnd": "^5.0.0",
-    "react-dnd-html5-backend": "^3.0.2",
-    "react-dom": "^16.3.1",
+    "react-dom": "^17.0.1",
```

Comment:
Remove this version

Reply:
in the morning, you told to use v17, right

Reply:
I told I needed to downgrade to 16. I think while resolving conflicts you copied your own package.json file from some branch and that's why my changes in the develop branch were overriden.